compiler/luci/pass/src/FuseInstanceNormPass.cpp

```diff
@@ -741,11 +731,12 @@ template <> bool InstanceNormPattern::match<InstanceNormPattern::PatternVersion:
   CHECK_OR_FALSE(rsqrt == rsqrt_should_be);
   CHECK_OR_FALSE(mean_of_ifm == mean_of_ifm_should_be);
 
-  // mul_gamma is absent
-  // const_as_gamma assume to be 1.0
+  // create 1.0 gamma and 0.0 beta
   auto graph = add_as_terminal->graph();
   const_as_gamma = make_const_one(graph, 1.0f);
   const_as_gamma->name(add_as_terminal->name() + "/gamma");
+  const_as_beta = make_const_one(graph, 0.0f);
+  const_as_beta->name(add_as_terminal->name() + "/beta");
```

Comment:
I think these lines are unnecessary. *1 and +0 are Nop.

And, `apply` functions for version_5 and version_6 have to be updated too. There are no gamma and beta for those patterns, so `reshape_gamma_beta` function should not be called.

Reply:
> And, apply functions for version_5 and version_6 have to be updated too.

I cannot check for ver 5 as of now and do not want to mix for both version in one PR.